packages/flutter/lib/src/material/chip.dart

```diff
@@ -74,6 +78,29 @@ class Chip extends StatelessWidget {
   /// The delete button is included in the chip only if this callback is non-null.
   final VoidCallback onDeleted;
 
+  /// The style to be applied to the chip's label.
+  ///
+  /// This only has effect on widgets that use a [TextStyle], such as [Text].
```

Comment:
A better way to phrase this would be "`...that respect the [DefaultTextStyle], such as [Text]`". I mean, `RichText` uses `TextStyle` too, but it won't affect that.